artemis-server/src/main/java/org/apache/activemq/artemis/core/transaction/Transaction.java

```diff
@@ -109,4 +109,8 @@ enum State {
 
    /** To be used on control transactions that are meant as internal and don't really require a hard sync. */
    Transaction setAsync(boolean async);
+
+   default boolean isAllowPageTransaction() {
+      return true;
+   }
```

Comment:
Similar confusion that this code default is true, but the overriding method looks to get a field value thats false by default?

Reply:
MirrorTransactions will disable page transaction by default. 

I have corrected the names